website/source/docs/builders/googlecompute.html.md

```diff
@@ -235,7 +261,10 @@ such support is implemented, startup scripts should be robust, as an image will
 when a startup script fails.
 
 ### Windows
-Startup scripts do not work on Windows builds, at this time.
+
+A Windows startup script can only be provided via the 'windows-startup-script-cmd' instance
```

Comment:
Windows startup script can be provided via the 'windows-startup-script-cmd', 'windows-startup-script-url', 'windows-startup-script-bat' and 'windows-startup-script-ps1' instance creation `metadata` field.

reference:- `https://cloud.google.com/compute/docs/startupscript#providing_a_startup_script_for_windows_instances`